src/base/preferences.h

```diff
@@ -139,6 +139,8 @@ class Preferences: public QObject
     void setScanDirsLastPath(const QString &path);
     bool isMailNotificationEnabled() const;
     void setMailNotificationEnabled(bool enabled);
+    QString getMailNotificationEmailSender() const;
```

Comment:
There are too many mails in "MailNotificationEmail"

Reply:
renamed to `getMailNotificationSender`